src/EnergyPlus/General.cc

```diff
@@ -325,20 +320,14 @@ void ProcessDateString(EnergyPlusData &state,
     // SUBROUTINE INFORMATION:
     //       AUTHOR         Linda Lawrie
     //       DATE WRITTEN   December 1999
-    //       MODIFIED       na
-    //       RE-ENGINEERED  na
 
     // PURPOSE OF THIS SUBROUTINE:
     // This subroutine will process a date from a string and determine
     // the proper month and day for that date string.
 
     // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-    int FstNum{};
-    bool errFlag{};
-    int NumTokens{};
-    int TokenDay{};
-    int TokenMonth{};
-    int TokenWeekday{};
```

Comment:
[src/EnergyPlus/General.cc:338]:(style),[variableScope],The scope of the variable 'NumTokens' can be reduced.